modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js

```diff
@@ -263,6 +263,21 @@ const CustomObjectFieldsList = ({keywords}) => {
 	const showCategories =
 		!!customFieldTypes.length && !!nativeFieldTypes.length;
 
+	const getDataDefinitionField = (fieldName) => {
+		const dataDefinitionField = dataDefinition.dataDefinitionFields.find(
+			(field) => field.name === fieldName
+		);
+
+		const settingsContext = dataLayoutBuilder.getDDMFormFieldSettingsContext(
+			dataDefinitionField
+		);
+
+		return {
+			...dataDefinitionField,
+			settingsContext,
+		};
+	};
+
```

Comment:
You probably missed this during the rebase, but this is no longer necessary with the changes of Aline and Milton, it is now called directly in the DataConverter.

Reply:
Yes, it's unused. Removing!